docs/contribution-guide/verification.md

```diff
@@ -0,0 +1,27 @@
+# Verification Instructions
+
+The purpose of this document is to provide a detailed explanation of how to verify a new released version of ```morphir-elm```. The following instructions are to be followed after every release in order to make sure that the release is valid.
+
+### Who this Guide is Designed For
+1. Business users of Morphir who would want to update the morphir-elm version to the latest release
+2. Developers who would want to verify the validity of a release
+
+### Upgrading ```morphir-elm``` to the latest version
+
+- Run ```npm install morphir-elm``` to upgrade to the latest version. This is the only step you need as a business user to upgrade to a new release of ```morphir-elm```
+
+### Verifying a ```morphir-elm``` release 
+
+After upgrade run the following commands to verify a release. The model to use as input can be found at ```morphir-elm\tests-integration\reference-model```.
```

Comment:
Be more specific about how to find the reference model. Or alternatively just describe how to build a minimal model. Or maybe provide a zip that contains one.